src/run.ts

```diff
@@ -181,20 +182,16 @@ export async function runVersion({
     let [versionCommand, ...versionArgs] = script.split(/\s+/);
     await exec(versionCommand, versionArgs, { cwd });
   } else {
-    let changesetsCliPkgJson = await require(require.resolve(
-      "@changesets/cli/package.json",
-      { paths: [cwd] }
```

Comment:
Why not just call `fs.realpathSync` just like it does and avoid the dependency?
https://github.com/sindresorhus/resolve-from/blob/4f02820afe9b4e9c379f4bb2745debdb49087d4f/index.js#L16

Reply:
Same reason as mentioned in #67: tests and robustness. This module is small and is well-used and well tested in the npm ecosystem (it's already present in `yarn.lock` here, for instance). Also the full code from `resolve-from` is:

```javascript
let fromDirectory = cwd
try {
  fromDirectory = fs.realpathSync(fromDirectory)
} catch (error) {
  if (error.code === 'ENOENT') {
    fromDirectory = path.resolve(fromDirectory)
  } else {
    throw error
  }
}
```